src/dotnet-roslyn-tools/PRTagger/PRTagger.cs

```diff
@@ -50,92 +46,97 @@ internal class PRTagger
             vsCommitSha, vsRepository.Id, cancellationToken: cancellationToken).ConfigureAwait(false);
         var previousVsCommitSha = vsCommit.Parents.First();
 
-        // Figure out which repo we're working with
-        if (!TryGetProductInfo(productName, out var productInfo))
+        foreach (var product in VSBranchInfo.AllProducts)
         {
-            logger.LogError($"Product info not found for {productName}.");
-            return -1;
-        }
+            // Get associated product build for current and previous VS commit SHAs
+            var currentBuild = await TryGetBuildNumberForReleaseAsync(product.ComponentJsonFileName, product.ComponentName, vsCommitSha, devdivConnection).ConfigureAwait(false);
+            var previousBuild = await TryGetBuildNumberForReleaseAsync(product.ComponentJsonFileName, product.ComponentName, previousVsCommitSha, devdivConnection).ConfigureAwait(false);
 
-        // Get associated product build for current and previous VS commit SHAs
-        var currentBuild = await TryGetBuildNumberForReleaseAsync(productInfo, vsCommitSha, devdivConnection).ConfigureAwait(false);
-        var previousBuild = await TryGetBuildNumberForReleaseAsync(productInfo, previousVsCommitSha, devdivConnection).ConfigureAwait(false);
+            var gitHubRepoName = product.RepoBaseUrl.Split('/').Last();
```

Comment:
Should check if this actually starts with "github.com", or have a boolean flag on Product that controls whether PRs will automatically be tagged. 

eg TypeScript has a `Product` but its not a github repo

Reply:
Good call, thanks!